lib/duty_calculator/calculation.rb

```diff
@@ -37,12 +38,17 @@ def transform_params(params)
 
     def self.get(params={})
       transformed_params = transform_params(validate_params(params))
+      # initialize connection
       conn = DutyCalculator::Client.new
+
       resp = conn.get "#{DutyCalculator::Client.api_base}/calculation", transformed_params
       hashed_resp = DutyCalculator::Response.new(resp.to_hash)
       resp = Hashie::Mash.new(hashed_resp)
-      raise Exception, "Duty Calculator Error: #{DutyCalculator::ErrorMessages.for_code(resp.body.error.code)}" if resp.body.error
-      raise Exception, "HTTP Status Code #{resp.status}" if resp.status.to_i != 200
```

Comment:
I did all i could to try and get them to return anything other than something that was handled by the first error and it didn't work... so i removed this